industrial_ci/ci_main.sh

```diff
@@ -321,10 +321,14 @@ travis_time_start after_script
 ## BEGIN: travis' after_script
 PATH=/usr/local/bin:$PATH  # for installed catkin_test_results
 PYTHONPATH=/usr/local/lib/python2.7/dist-packages:$PYTHONPATH
+# For compatibilility with hydro catkin, which has no --verbose flag
```

Comment:
A matter of taste, but we're trying to define (environment) variables earlier as possible and collect at fewer places (around [here](https://github.com/ros-industrial/industrial_ci/blob/6f2e70627bfa86bd7fab8c6821d6ddfe35f3314c/industrial_ci/ci_main.sh#L87)). Would you want to move this there too?